Studentify/Models/StudentifyAccount.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Studentify.Models.Authentication;
+
+namespace Studentify.Models
+{
+    /// <summary>
+    /// Class that represents app user's account model in database.
+    /// </summary>
+    public class StudentifyAccount
+    {
+        [Key] 
+        public int Id { get; set; }
+        public string StudentifyUsername { get; set; }
+    }
```

Comment:
Można dodać jakieś imie, nazzwisko i co tam chcemy jeszcze na profilu wyświetlać

Reply:
Dodałem imię, Nazwisko i Maila. Po ustaleniu z analitykami można to poszerzyć ale teraz raczej nam wystarczy do podstawowych działań.